dtfs-central-api/src/v1/controllers/portal/deal/delete-deal.controller.ts

```diff
@@ -20,10 +23,11 @@ export const deleteDeal = async (req: CustomExpressRequest<{ params: { id: strin
     if (error instanceof InvalidAuditDetailsError) {
```

Comment:
My code oops but you could consolidate both try/catches and check for `ApiError` rather than `invalidAuditDetailsError` then `DocumentNotDeletedError`

Reply:
/ once alexs pr is in can do `next(error)` too